src/Controller/WelcomeAction.php

```diff
@@ -31,7 +32,7 @@ public function __invoke(Request $request): Response
             return new Response($this->twig->render('welcome.html.twig'));
         }
         if (false === \filter_var($pimUrl, FILTER_VALIDATE_URL)) {
-            throw new \LogicException('PIM url is not valid.');
+            throw new BadRequestHttpException('PIM url is not valid.');
```

Comment:
Is the error explicitly displayed ? Because it may be a good point to explain why it fails.

Reply:
not realy 
![image](https://user-images.githubusercontent.com/7101819/163354260-4e5b6d64-006f-4822-805c-6a8c644c5a36.png)

Reply:
We should have a better page for 400 errors to include the exception message.

Reply:
I could fix it in this PR...?

Reply:
Honestly, you can, it's easy, you only have to create the `error400.html.twig` like we already did for the 400 and display the error message in it.

Reply:
@masacc what did you have in mind for a more explicit message?

Reply:
exactly what @tseho said :smile_cat:

Reply:
@masacc @tseho :arrow_up: